edi/tests/test_edi_mixin.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2020 Creu Blanca
+# @author: Enric Tobella
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import mock
+
+from odoo import api, models
+
+from .common import EDIBackendCommonTestCase
+
+
+class EdiExchangeRecordMixinTest(models.Model):
```

Comment:
never define test models in a model that is imported at runtime as it gets there forever.
Please use odoo-test-helper (yeah, brand new), Example https://github.com/OCA/search-engine/blob/13.0/connector_search_engine/tests/test_all.py#L21

Reply:
Thanks!!! I was looking for this code but I was unable to find it :smile: